src/diffusions.jl

```diff
@@ -415,7 +413,7 @@ function pagerank(A,alpha::Float64)
 end
 
 function pagerank(A,alpha::Float64,tol::Float64)
-    _personalized_pagerank_validated(A,alpha,1./size(A,1),tol)
+    _personalized_pagerank_validated(A,alpha,1. /size(A,1),tol)
```

Comment:
I think you can write these as 1.0/size(A,1)

Reply:
Sure... Changed. 2488e36adc7a947be8cba864c5d175c95263bf12